src/javascript/app/base/binary_loader.js

```diff
@@ -141,19 +141,18 @@ const BinaryLoader = (() => {
             loadActiveScript(config);
         }
         if (config.no_mf && Client.isLoggedIn() && Client.isAccountOfType('financial')) {
-            BinarySocket.wait('authorize').then((res) => {
-                if (Client.isOfferingBlocked(res.authorize.country)) {
-                    displayMessage(error_messages.offerings_blocked());
-                } else {
-                    displayMessage(error_messages.no_mf());
-                }
+            BinarySocket.wait('authorize').then(() => {
+                displayMessage(error_messages.no_mf());
```

Comment:
Possible to move the entire if statement inside the `wait` call back on line 149?

Reply:
yes, sure